### PR TITLE
build: build renovate docker images on release

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Build our renovate images after the npm release

## Context
requires those to be merged before:
- https://github.com/renovatebot/docker-renovate/pull/2118
- https://github.com/renovatebot/docker-renovate-full/pull/2650

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/workflows/release.yml
Comment (rarkins) [reply .github/workflows/release.yml:120]: This repo has access to the same organization secrets for docker env. It should also have write access to the two packages.

Reconnect the latter how? Is that something I need to do now?

Comment (viceice) [reply .github/workflows/release.yml:120]: we should add a dummy cache repo for this, so the cache images are not shown on main repo packages. I've done the same on containerbase[^1]
go to `docker-build-cache` package settings removed the repo source and add the `docker-build-cache` renovate repo as new source

[^1]: https://github.com/containerbase/docker-build-cache

**Review by secustor — COMMENTED** [review]
LGTM so far

**Comment by rarkins** [comment]
Do you anticipate having any "race conditions" where you publish version x.y.z to npmjs and then the corresponding Docker image fails to install the same version because the npmjs CDN says it's not available?

Is it already the case that every version we publish to npm will have a corresponding version on Docker? i.e. GitHub doesn't cancel duplicate workflows, and if a Docker publish fails after npm publish succeeds, is there a way to recover?

Do we have a nice way to roll forward and back with this approach after merge if we find problems?

**Comment by viceice** [comment]
> Do you anticipate having any "race conditions" where you publish version x.y.z to npmjs and then the corresponding Docker image fails to install the same version because the npmjs CDN says it's not available?

🤔 Didn't thought about that yet.

> 
> Is it already the case that every version we publish to npm will have a corresponding version on Docker? i.e. GitHub doesn't cancel duplicate workflows, and if a Docker publish fails after npm publish succeeds, is there a way to recover?

We don't publish all versions to docker yet, because the current images are not updated that often and we only rebuild last version.

> 
> Do we have a nice way to roll forward and back with this approach after merge if we find problems?

Currently the docker images are not published. I've disabled to just test the docker build on release.
If build succeeds fine, the we can enable image publishing in a new PR. If build fails because of npmjs issues, then we can revert / or fix the build first by another approach. (not using npmjs renovate, but using a local build)

**Review by rarkins — DISMISSED** [review]

File: .github/workflows/release.yml
Comment (viceice) [reply .github/workflows/release.yml:127]: yes, will be enabled on seperate pr

**Review by secustor — APPROVED** [review]

**Pull request merged**
